website/content/docs/audit/index.mdx

```diff
@@ -74,20 +74,20 @@ The existing logs that it did store are untouched.
 
 ## Blocked Audit Devices
 
-If there are any audit devices enabled, Vault requires that at least
-one be able to persist the log before completing a Vault request.
+If the audit device request becomes blocked for an indefinite period of time,
+then the associated Vault request will block as well.
 
-!> If you have only one audit device enabled, and it is blocking (network
-block, etc.), then Vault will be _unresponsive_. Vault **will not** complete
-any requests until the audit device can write.
-
-If you have more than one audit device, then Vault will complete the request
-as long as one audit device persists the log.
+!> Note that even with multiple audit devices enabled, if an audit device
+becomes blocked (due to network issue or for other reasons), then Vault will become
+_unresponsive_. Vault **will not** complete any requests until the blocked
+audit device can write.
 
 Vault will not respond to requests if audit devices are blocked because
 audit logs are critically important and ignoring blocked requests opens
-an avenue for attack. Be absolutely certain that your audit devices cannot
-block.
+an avenue for attack.
```

Comment:
I know you didn't write this paragraph but it feels disingenuous to me.  I believe the reason for the current behaviour is because we don't know how to fix it, or we haven't taken the time, not because this is a security measure.  If that were the case, we would apply the same policy to non-blocking audit device failures.

Or maybe this is an unfortunate use of the word "blocking" and what the paragraph really means to say is: Vault will not respond to requests when no audit devices can record them, because
audit logs are critically important and ignoring auditing failures opens an avenue for attack.